site/docs/pages/guides/telemetry.mdx

```diff
@@ -0,0 +1,80 @@
+---
+title: Telemetry · OnchainKit
+description: Understanding OnchainKit's anonymous telemetry system and how to configure it.
+---
+
+## Telemetry
+
+OnchainKit is introducing an anonymous telemetry system to help us better understand how our library is used in the wild. Participation in this anonymous program is optional—if you'd prefer not to share any usage data, you can easily opt out.
+
+### Why Are We Collecting Telemetry?
+
+OnchainKit has quickly become a go-to full‑stack component library for integrating essential onchain functionality (like `<wallet />`, `<transaction />`, and `<mintNFT />`) in minutes. Until now, our usage insights have been limited to public npm download counts and API endpoint usage. By collecting telemetry data, we can:
+
+- **Gauge Component Usage**: Understand which components (and their variants) are most popular
+- **Data-Informed Improvements**: Help our data science team generate insights that drive future enhancements and refactoring decisions
+- **Proactive Monitoring**: Quickly detect issues with new releases or API changes through a dedicated error event stream (with alerts to oncall engineers)
+
+### What Data Will Be Collected?
+
+Telemetry data is completely anonymous and designed to provide aggregated insights. Specifically, we collect:
+
+- **Command Details**: Which commands (or component events) are being invoked (e.g. walletConnection, swapSuccess)
+- **Version & App Info**: The OnchainKit version, app name (from window.top.document.title), and origin (the app URL)
+- **Usage Metrics**: Information such as the number of unique wallets, transactions, or contracts interacting with OnchainKit
+- **Error Events**: Generic error events along with component context to help us triage any issues
```

Comment:
is it possible to provide exactly the data we collect? eg. like the specific form of the JSON? 

reading this my first thought is "these are just words, what is actually happening?"

Reply:
perhaps that's the example telemetry event below, in which case, i would say somethiing here like, "see the next section for specifics" or "read on for more details"